frontend/src/components/Home.tsx

```diff
@@ -0,0 +1,76 @@
+import { Button, Container, Stack } from '@chakra-ui/react';
+import { useState }                 from 'react';
+import BaseHelper                   from '../helpers/BaseHelper';
+
+const Home = () => {
+    const [getState, setState] = useState({
+        showInstructions: false
+    })
+    return (
+        <div>
+            <Container>
+                <Stack spacing={2}>
+                    <h1>
+                        Dobrodošli na spletni strani Secret Santa
+                    </h1>
+                    <Button onClick={() => BaseHelper.toggleCheckbox(getState, setState, 'showInstructions')}>
+                        {
+                            !getState.showInstructions
+                                ? <h1>Pokaži navodila</h1>
```

Comment:
You could just do `<h1>{ getState.showInstructions ? 'Skrij' : 'Pokaži' } Navodila</h1>`